packages/server-wallet/src/wallet/store.ts

```diff
@@ -14,19 +13,23 @@ import {
 import _ from 'lodash';
 import {Either, right} from 'fp-ts/lib/Either';
 import {Bytes32, ChannelResult} from '@statechannels/client-api-schema';
+import Objection from 'objection';
 
 import {Channel, SyncState, RequiredColumns} from '../models/channel';
 import {SigningWallet} from '../models/signing-wallet';
 import {addHash} from '../state-utils';
 import {ChannelState} from '../protocols/state';
+import knex from '../db/connection';
 
 export const Store = {
+  transaction: <T>(callback: (transaction: Objection.Transaction) => Promise<T>): Promise<T> =>
+    knex.transaction(callback),
```

Comment:
If I am reading this code correctly, this is a single parameter function that calls another single parameter function without modifying the parameter. But perhaps the syntax above a more readable type signature?
```suggestion
  transaction: knex.transaction,
```

Reply:
That's a good point, I think we just need ```transaction: knex.transaction```